semantic_conventions/trace/rpc.yaml

```
@@ -78,3 +78,33 @@ groups:
           required: always
           brief: "The [numeric status code](https://github.com/grpc/grpc/blob/v1.33.2/doc/statuscodes.md) of the gRPC request."
           examples: [0, 1, 16]
+    - id: rpc.jsonrpc
+      prefix: rpc.jsonrpc
+      extends: rpc
+      brief: 'Tech-specific attributes for [JSON RPC](https://www.jsonrpc.org/).'
+      attributes:
+        - id: version
+          type: string
+          required:
+            conditional: 'If missing, it is assumed to be "2.0".'
+          brief: "Protocol version as in `jsonrpc` property of request/response. Since first protocol version does not specify this, value can be omitted entirely."
```

```suggestion
          required:
            conditional: 'If missing, it is assumed to be "1.0".'
          brief: "Protocol version as in `jsonrpc` property of request/response. Since JSON-RPC 1.0 does not specify this, the value can be omitted."
```

I just noticed that this is not consistent. If it is missing, `1.0` would have to be assumed, right?